src/test/java/uk/gov/ons/ctp/response/action/scheduled/distribution/ActionDistributorTest.java

```diff
@@ -117,6 +117,18 @@ public void testFailToGetAnyAction() throws Exception {
     verify(actionProcessingService, times(0)).processActionCancel(any(Action.class));
   }
 
+  /**
+   * We retrieve actionTypes but then exception thrown when retrieving actions.
+   *
+   * @throws Exception oops
```

Comment:
If the test method was `testExceptionNotThrowWhenNoActionTypesLeads` this comment would probably be redundant. Also `@throws Exception oops` is pointless :)

Reply:
copy/paste problems... cheers 👍